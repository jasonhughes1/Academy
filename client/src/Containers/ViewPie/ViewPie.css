.view-pie-container {
  display: flex;
  flex-direction: column;
  background-color: #3C4040;
}

.place-holder {
  height: 100px;
}
