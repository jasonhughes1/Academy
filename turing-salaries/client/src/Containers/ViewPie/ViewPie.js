import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';
import { connect } from 'react-redux';
import Pie from '../Pie/Pie';

class ViewPie extends Component {

  render() {
    return (
      <div className='view-salary-container'>
        <div className='link-container-view'>
        <NavLink className='nav' exact to='/'>Home</NavLink>
      </div>
        <h1 className='view-salary-title'>Salary and Job Comparisons</h1>
          <Pie />
      </div>
    )
  }
}

export const mapStateToProps = (store) => {
  return {
    alumni: store.alumData
  };
};

export default connect(mapStateToProps, null)(ViewPie);
