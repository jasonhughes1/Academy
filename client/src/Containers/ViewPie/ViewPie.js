import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';
import { connect } from 'react-redux';
import Pie from '../Pie/Pie';
import './ViewPie.css';

export class ViewPie extends Component {

  render() {
    return (
      <div className='view-pie-container'>
        <div className='link-container-view'>
        <NavLink className='nav' exact to='/'>Home</NavLink>
      </div>
        <h2 className='view-salary-title'>Turing grads have been accepting offers with these job titles</h2>
          <Pie />
          <div className='place-holder'></div>
      </div>
    )
  }
}

export const mapStateToProps = (store) => {
  return {
    alumni: store.alumData
  };
};

export default connect(mapStateToProps, null)(ViewPie);
