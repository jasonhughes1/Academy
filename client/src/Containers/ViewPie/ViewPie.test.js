import React from 'react';
import { shallow } from 'enzyme';
import { ViewPie, mapStateToProps } from './ViewPie';


describe('ViewPie Tests', () => {

  it('should be defined', () => {
    const renderedViewPie = shallow(<ViewPie alumni= {[]}/>);

    expect(renderedViewPie).toBeDefined();
  });

  it('should match the snap shot', () => {
    const renderedViewPie = shallow(<ViewPie alumni= {[]}/>);

    expect(renderedViewPie).toMatchSnapshot();
  });
  
});
