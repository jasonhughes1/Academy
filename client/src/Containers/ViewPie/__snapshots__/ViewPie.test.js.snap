// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewPie Tests should match the snap shot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ViewPie
    alumni={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="link-container-view"
>
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    className="nav"
                    exact={true}
                    to="/"
          >
                    Home
          </NavLink>
</div>,
        <h2
          className="view-salary-title"
>
          Turing grads have been accepting offers with these job titles
</h2>,
        <Connect(Pie) />,
        <div
          className="place-holder"
/>,
      ],
      "className": "view-pie-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="nav"
            exact={true}
            to="/"
>
            Home
</NavLink>,
          "className": "link-container-view",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Home",
            "className": "nav",
            "exact": true,
            "to": "/",
          },
          "ref": null,
          "rendered": "Home",
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Turing grads have been accepting offers with these job titles",
          "className": "view-salary-title",
        },
        "ref": null,
        "rendered": "Turing grads have been accepting offers with these job titles",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "place-holder",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="link-container-view"
>
            <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        className="nav"
                        exact={true}
                        to="/"
            >
                        Home
            </NavLink>
</div>,
          <h2
            className="view-salary-title"
>
            Turing grads have been accepting offers with these job titles
</h2>,
          <Connect(Pie) />,
          <div
            className="place-holder"
/>,
        ],
        "className": "view-pie-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              ariaCurrent="true"
              className="nav"
              exact={true}
              to="/"
>
              Home
</NavLink>,
            "className": "link-container-view",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": "Home",
              "className": "nav",
              "exact": true,
              "to": "/",
            },
            "ref": null,
            "rendered": "Home",
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Turing grads have been accepting offers with these job titles",
            "className": "view-salary-title",
          },
          "ref": null,
          "rendered": "Turing grads have been accepting offers with these job titles",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "place-holder",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
